science/libsbmlsim/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        libsbmlsim libsbmlsim 1.4.0 v
+name                libsbmlsim
```

Comment:
Missed this before. The `name` line can be removed because the github portgroup will name it that already - it is the 2nd parameter of `github.setup`.

Reply:
I also removed the `name` line. Both fixes are squashed and now it is in a single commit.